core/src/main/scala/org/apache/spark/TaskContext.scala

```diff
@@ -150,6 +150,10 @@ abstract class TaskContext extends Serializable {
    */
   def stageId(): Int
 
+  /**
+   * The attempt ID of the stage that this task belongs to.
```

Comment:
can we follow `taskAttemptId` and say more in the document?

Reply:
Of course, I can add more doc.

But is it necessary? I think the field(function) name is already self-explanatory.